build.conf

```diff
@@ -36,18 +36,15 @@ SQLITE_VERSION="3460000"
 BOOTSTRAP_GET_PIP="https://bootstrap.pypa.io/get-pip.py"
 
 # Python modules installed after bootstraping pip.
-PIP_VERSION="24.0"
-SETUPTOOLS_VERSION="70.0.0"
+PIP_VERSION="24.1.2"
+SETUPTOOLS_VERSION="71.0.3"
 # pycparser is explicitly installed to work around setuptools auto dependencies.
 PYCPARSER_VERSION="2.22"
 
 # Python modules that have to be built and/or installed in Pythia.
-PSUTIL_VERSION="5.9.8"
+PSUTIL_VERSION="6.0.0"
```

Comment:
Do we still need this variabl here.

I think that we can install all the packageis via `pip install -r requirements.txt`

then the `requirements.txt` file will be used by the security scanners.

Reply:
Hhmmm… Now that you put it this way, it's obvious `pywin32` is missing from `requirements.txt`. Unless the dev building locally uses Windows, of course…

There's also `pip` itself, which is installed in a peculiar way, which differs on Windows compared to the other OS'es.

Actually, lately only `psutil` is always installed in a generic way, using the the still present `PIP_LIBRARIES` env var. And this is only until https://github.com/chevah/pythia/issues/60 is fixed.

So, I guess no?

Reply:
> Hhmmm… Now that you put it this way, it's obvious `pywin32` is missing from `requirements.txt`. Unless the dev building locally uses Windows, of course!

I guess this could be fixed through https://github.com/stefanzweifel/git-auto-commit-action to automatice updating `requirements.txt` through the Windows GitHub Action. Of course, only after there's a `pywin32` wheel for Python 3.13 and the Windows build can finish successfully.

Reply:
Also filled https://github.com/chevah/pythia/issues/62 for improving current situation.